src/VisualStudio/Core/Next/Remote/ServiceHubRemoteHostClient.WorkspaceHost.cs

```diff
@@ -16,56 +17,49 @@ internal partial class ServiceHubRemoteHostClient
         private class WorkspaceHost : ForegroundThreadAffinitizedObject, IVisualStudioWorkspaceHost, IVisualStudioWorkingFolder
         {
             private readonly VisualStudioWorkspaceImpl _workspace;
-            private readonly RemoteHostClient _client;
+
+            /// <summary>
+            /// The current session we have open to the remote host.  Only accessible from the
+            /// UI thread.
+            /// </summary>
+            private readonly KeepAliveSession _session;
```

Comment:
since it will live until VS shutdown. you don't need to. I meant if you want to explicitly shutdown, you can call shutdown. but in this particular case, I dont think you need to. it will automatically shutdown when VS process goes away.